.github/workflows/run-pytest.yml

```diff
@@ -48,4 +49,4 @@ jobs:
           DB_USERNAME: ${{ secrets.DB_USERNAME }}
           DB_PASSWORD: ${{ secrets.DB_PASSWORD }}
         run: |
-          pytest
+          pytest --splits 5 --group ${{ matrix.group }}
```

Comment:
The number of splits should match the number of groups here.

Reply:
Oh good catch.